src/App.css

```
@@ -58,7 +62,15 @@
   padding: 8px;
   border-radius: 7px;
   margin:20px;
-  height:30vh;
+  height:60vh;
```

you might be able to use something like this, so that the height can just be to height of what ever the content is:


```suggestion
  height:fit-content;
```